### DISCUSSION TITLE
Issue with geospatial index

### DISCUSSION THREAD

**Discussion by Rik0253** [jenssegers/laravel-mongodb#2122] [Q&A]
I am getting 'Cant extract geo keys' 
Using laravel version -7.x
jessenger/mongodb-3.7.0
mongodb/mongodb-1.6.1
php version - 7.4.2

I am getting the coordinates from google place api. And storing it as 
$property->property_location=['type'='Point','coordinates'=>[88.3030967,22.4742253]];
$property->save();

what i am doing wrong?

Thanks in advance!

**Comment by divine** [comment]
I've answered your question, why are you spamming and wasting time?

**Reply by Rik0253** [reply]
Sorry @divine I have no intension of spamming . It was by mistake posted you can delete the post and thank you for guidance.
